docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -428,3 +435,86 @@ export default clerkMiddleware(
 
 > [!NOTE]
 > Dynamic keys are not accessible on the client-side.
+
+### OrganizationSyncOptions
+
+The `organizationSyncOptions` property on the <code>[clerkMiddleware()](#clerk-middleware-options)</code> options
+object has the type `OrganizationSyncOptions`.
+
+This defines the options for automatically activating an organization or the [personal account](/glossary#personal-account)
+based on the URL. This is useful for applications that require organization slugs or IDs to appear in URL paths.
+
+If a mismatch between the active organization on the session (e.g. as reported by
+<code>[auth()](/docs/references/nextjs/auth)</code>) and the organization as indicated by the URL is detected, an attempt
+to activate the given organization will be made.
+
+`OrganizationSyncOptions` has the following properties:
+
+<Properties>
+  - `organizationPatterns`
+  - <code>[Pattern](#pattern)\[]</code>
+
+  URL patterns that are organization-specific and contain an organization ID or slug as a path parameter. If a request
+  matches this path, the organization identifier will be extracted and activated before rendering.
+
+  If the route also matches the personalAccountPatterns, this takes precedence.
```

Comment:
🔧 Can we have `personalAccountPatterns` appear with in-line code formatting, or even link to the definition?

Reply:
👍 

I don't see a precedent or a way to link to properties, but I can do the code formatting!

ff0d75ef3631d2f2665bd9508eb9af4a00e5a2db